python_modules/dagster-webserver/dagster_webserver/webserver.py

```diff
@@ -226,6 +229,9 @@ def index_html_endpoint(self, request: Request):
             with open(index_path, encoding="utf8") as f:
                 rendered_template = f.read()
                 nonce = uuid.uuid4().hex
+                mimetypes.add_type('application/javascript', '.js')
+                mimetypes.add_type('text/css', '.css')
+                mimetypes.add_type('image/svg+xml', '.svg')
```

Comment:
Looks like this works, but I think it may not be quite the right place for this, since this is where we're serving the index file.

Can you move these calls to `build_static_routes`, so that they're alongside where we initialize the relevant static paths themselves?

Reply:
@hellendag Done! Is that better?